vtex/address-form
# CHK-482: Add rules for ARE and fix ITA number with geolocation

#### What is the purpose of this pull request?

This adds specific rules for United Arab Emirates, and tweaks the rules for Italy.

#### What problem is this solving?

Fixes geolocation usage in both countries, where if you inputted an address without a number the user wouldn't be able to proceed from the shipping step.

#### How should this be manually tested?

[Workspace](https://lucas--vtexgame1geo.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&sc=2).

You can use the following addresses:

 - United Arab Emirates: W Corniche Rd - Al Ras Al Akhdar.
 - Italy: Via del Telegrafo.

For both addresses you should be able to input a number and move forward.

#### Screenshots or example usage

#### Types of changes
